src/pages/governance/[...slug].tsx

```diff
@@ -35,9 +35,7 @@ export const getServerSideProps: GetServerSideProps<Props> = async ({ query, par
 
   const id = params?.slug;
   const proposalID = Array.isArray(id) ? id[0] : id;
-  const chainId =
-    Number(query.chainId as string) ||
-    (process.env.NODE_ENV === 'production' ? Celo.id : Alfajores.id);
+  const chainId = Number(query.chainId as string) || Celo.id;
```

Comment:
everywhere else we default to Celomainnet not sure why we (I) default to Alfajores only when on Dev mode here.